pkg/openstack/loadbalancer_test.go

```diff
@@ -5,10 +5,12 @@ import (
 	"sort"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
Let's not place the declaration of assert module, ideally all third party deps should be listed together like they had been earlier.

Reply:
+1, I don't understand why this import is moved.